website/common/locales/en/tasks.json

```diff
@@ -178,5 +178,6 @@
     "yesterDailiesCallToAction": "Start My New Day!",
     "yesterDailiesOptionTitle": "Confirm that this Daily wasn't done before applying damage",
     "yesterDailiesDescription": "If this setting is applied, Habitica will ask you if you meant to leave the Daily undone before calculating and applying damage to your avatar. This can protect you against unintentional damage.",
-    "repeatDayError": "Please ensure that you have at least one day of the week selected."
+    "repeatDayError": "Please ensure that you have at least one day of the week selected.",
+    "sessionOutdated": "Your session is outdated. Please reload the page."
```

Comment:
Given this comes from the API, the user might not be interacting with a "page" at all. Maybe "Please refresh or sync"?